Collector/ProfilePlugin.php

```diff
@@ -58,7 +58,9 @@ public function handleRequest(RequestInterface $request, callable $next, callabl
     {
         $profile = new Profile($this->pluginName, $this->formatter->formatRequest($request));
 
-        $this->collector->getCurrentStack()->addProfile($profile);
+        if (false !== $stack = $this->collector->getCurrentStack()) {
```

Comment:
Silently fail will just hide details in the profiler, but it may be hard for users to find why my plugin is not in the profiler.

Throwing a LogicException may do the job unless I missed some cases while refactoring the profiler.